applications/blackhole/src/blackhole_socket_callback.erl

```diff
@@ -39,6 +39,12 @@ recv(_SessionPid, _SessionId, {'event', _Ignore, <<"subscribe">>, SubscriptionJO
     end,
     {'ok', Context1};
 
+recv(SessionPid, SessionId, {'event', _Ignore, <<"clear_subscriptions">>, _Data}, Context) ->
```

Comment:
@onnet i like this, how about changing it to "unsubscribe" and process the _Data object sent ?
the _Data object can have unsubscribe info for account_id, listof account_id, all (or empty | null _Data).
thoughts ?